documentation/src/main/asciidoc/topics/con_server_hinting.adoc

```diff
@@ -1,7 +1,24 @@
 [id='server-hinting_{context}']
 = Server hinting
 
-The following topology hints can be specified:
+Server hinting helps you to achieve high availability of your data in case of hardware and network failure.
+When you configure server hinting, {brandname} distributes your data across the cluster and stores backup copies of your data on as many servers, racks, and data centers as possible.
+
+NOTE: When you use replicated caches, {brandname} keeps a complete copy of all entries on each node. Server hinting distributes data across the topology so that backup copies of data are stored on different nodes in the topology than the primary owners.
+
+{brandname} uses consistent hashing mechanism to distribute data across nodes.
+The algorithm can potentially move a greater number of segments than necessary during cluster rebalancing.
+You can mitigate this by configuring a larger number of segments.
+However, using a large number of segments can lead to uneven distribution of segments.
```

Comment:
```suggestion
Server hinting can impact cluster rebalancing operations by moving more segments than necessary if the number of segments for the cache is too low.
Increasing the number of segments too much can lead to uneven distribution.
```

Reply:
I don't think `Increasing the number of segments too much can lead to uneven distribution` is true. I don't think there is any real side effect of having a large number of segments except for resources consumption.

Reply:
Thanks @pruivo 
What about?
```suggestion
Server hinting can impact cluster rebalancing operations by moving more segments than necessary if the number of segments for the cache is too low.
Increasing the number of segments requires available compute resources.
```
And is there any downside to server hinting that we should mention? Something we're missing in here. Thanks

Reply:
looks good to me.

> And is there any downside to server hinting that we should mention? Something we're missing in here. Thanks

Nothing worth mention IMO. It may take more time to compute the mapping between segment -> nodes than the default, but you won't notice in modern machines :)